package.json

```
@@ -1,4 +1,6 @@
 {
+  "name": "twill",
```

Perhaps will be better to scope the package for possible future usage?

```suggestion
  "name": "@area17/twill",
```